contracts/src/main/kotlin/com/r3/developers/csdetemplate/utxoexample/states/ChatState.kt

```diff
@@ -24,7 +24,11 @@ data class ChatState(
     // The message
     val message: String,
     // The participants to the chat, represented by their public key.
-    override val participants: List<PublicKey>) : ContractState {
+    private val participants: List<PublicKey>) : ContractState {
+
+    override fun getParticipants(): List<PublicKey> {
```

Comment:
is this change part of the Java API change ?

Reply:
Yes